docs/dispatch/Dispatch_and_You.md

```diff
@@ -16,3 +16,6 @@ Let's get you pointed in the right direction; follow this little Table of Conten
 
 > info
 > To use Dispatch, you will need an application id for your game. You should have gotten this id when you set yourself up to use the Discord GameSDK, but if not, ask a Discord team member to create an application for you and get you the id.
+
+> info
```

Comment:
This won't render properly in docs unfortunately. Two info blocks next to each other will just not show the second one. If you make this just a normal paragraph instead it'll be fine.